R/shiny-recorder.R

```diff
@@ -153,7 +153,7 @@ RecordingSession <- R6::R6Class("RecordingSession",
       private$localHost <- host
       private$localPort <- port
       private$outputFileName <- outputFileName
-      private$outputFile <- file(outputFileName, "w")
+      cat("", file = private$outputFileName, append = FALSE)
```

Comment:
The purpose of this is to create the file and also to error if it can't be written to?

Reply:
To initialize the file, yes.